.vale/styles/Vocab/Che/accept.txt

```diff
@@ -86,6 +87,7 @@ HTTPS|https
 I/O
 IDE|ide|IDEs
 Intelephense
+IntelliJ
```

Comment:
Consider adding IntelliJ to
https://61376cee05bf377748f25ba8--eclipse-che-docs-pr.netlify.app/che-7/end-user-guide/using-alternative-ides-in-che/ On the line "Completely different web IDEs, such as `IntelliJ`, Jupyter, Eclipse Dirigible, or others. "

Reply:
Done.